release-notes/opensearch-release-notes-2.0.0.md

```diff
@@ -0,0 +1,637 @@
+* Add depreciation notice to security tools ([#1756](https://github.com/opensearch-project/security/pull/1756))
+* [Practice] Reverting changes ([#1754](https://github.com/opensearch-project/security/pull/1754))
+* Renames securityconfig folder to config in bundle step and makes relevant changes ([#1749](https://github.com/opensearch-project/security/pull/1749))
+* Updated issue templates from .github. ([#1740](https://github.com/opensearch-project/security/pull/1740))
+* Updates Dev guide ([#1590](https://github.com/opensearch-project/security/pull/1590))
+* List out test failures in CI log ([#1737](https://github.com/opensearch-project/security/pull/1737))
+* Make Git ignore out/ directory ([#1734](https://github.com/opensearch-project/security/pull/1734))
+* Fix data-stream name resolution for wild-cards ([#1723](https://github.com/opensearch-project/security/pull/1723))
+* Remove support for JDK14 ([#1720](https://github.com/opensearch-project/security/pull/1720))
+* Speeding up tests ([#1715](https://github.com/opensearch-project/security/pull/1715))
+* Fix min_doc_count handling when using Document Level Security ([#1714](https://github.com/opensearch-project/security/pull/1714))
+* Set the mapped security roles of the user so these can be used by the DLS privileges evaluator. Allow security roles to be used for DLS parameter substitution. Fixes opensearch-project/security/#1568 ([#1588](https://github.com/opensearch-project/security/pull/1588))
+* Convert Plugin install to only build once ([#1708](https://github.com/opensearch-project/security/pull/1708))
+* Upgrade to Gradle 7 ([#1710](https://github.com/opensearch-project/security/pull/1710))
+* Move CodeQL into parallel workfow ([#1705](https://github.com/opensearch-project/security/pull/1705))
+* Seperate BWC tests into parallel workflow ([#1706](https://github.com/opensearch-project/security/pull/1706))
+* Fixes broken test due to unsupported EC using JDK-17 ([#1711](https://github.com/opensearch-project/security/pull/1711))
+* Centralize version settings ([#1702](https://github.com/opensearch-project/security/pull/1702))
+* Remove TransportClient auth/auth ([#1701](https://github.com/opensearch-project/security/pull/1701))
+* Add new code hygiene workflow ([#1699](https://github.com/opensearch-project/security/pull/1699))
+* Remove JDK8 from CI ([#1703](https://github.com/opensearch-project/security/pull/1703))
+* Add CI check for demo script ([#1690](https://github.com/opensearch-project/security/pull/1690))
+* Introduce BWC tests in security plugin ([#1685](https://github.com/opensearch-project/security/pull/1685))
+* Correct the step name in CI ([#1683](https://github.com/opensearch-project/security/pull/1683))
+* Add support for DLS Term Lookup Queries ([#1541](https://github.com/opensearch-project/security/pull/1541))
```

Comment:
This bullet point (Add support for DLS term lookup queries) should go under "FEATURES".